assets/js/common/anonymous-user.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+define(['require',
+        'angular',
+        'angularfire',
+        'common/firebase-service'], function(require) {
+
+    angular
+      .module('usersFactoryModule', [
+      'firebase',
+      'voteFireRefServiceModule'
+      ])
+      .factory("AnonymousUser",
```

Comment:
This should be just "User", because in future we are going to implement authentication.